client/views/session.tpl

```diff
@@ -0,0 +1,17 @@
+<p class="clearfix">
```

Comment:
I played a bit with the markup and it seems to me like you don't need `clearfix` here. At least I couldn't see any difference with / without at different resolutions.